### PR TITLE
Click integration

### PR DESCRIPTION
**Pull request by angelofenoglio**
## What?

 - Use `click` to implement CLI logic
 - Improve packaging
 - Create initial built in modules (subcommands)

### CODE REVIEW

**Review by exequielrafaela — COMMENTED** [review]
@diego-ojeda-binbash @angelofenoglio not sure if we'll finally attend all the pending updates detailed at https://github.com/binbashar/le-tf-infra-aws/issues/47#issuecomment-842450439

Thanks for all the great improvements already done! Let's keep moving forward 💪🏼

**Comment by diego-ojeda-binbash** [comment]
> @diego-ojeda-binbash @angelofenoglio not sure if we'll finally attend all the pending updates detailed at [binbashar/le-tf-infra-aws#47 (comment)](https://github.com/binbashar/le-tf-infra-aws/issues/47#issuecomment-842450439)
> 
> Thanks for all the great improvements already done! Let's keep moving forward 💪🏼

@exequielrafaela I think the only item that should addressed in this PR is "list tasks by default when build file is present and when it is not". The rest of the items should be part of a different ticket/scope.

+ @angelofenoglio

**Comment by angelofenoglio** [comment]
I agree with @diego-ojeda-binbash, it would be neater. Also this is already quite the big PR, we should strive for smaller, more atomic PRs from now on.

**Comment by angelofenoglio** [comment]
In that regard, I think I'll remove `modules/leverage.py` for now, to add it in the next PR when it is more mature and not only a draft. @diego-ojeda-binbash @exequielrafaela

**Comment by exequielrafaela** [comment]
@angelofenoglio @diego-ojeda-binbash Thanks for your quick responses, everything sounds good.

File: README.md [hunk 1]
```diff
@@ -26,16 +30,21 @@ First, make sure your Python version is 3.6 or higher. Then install via pip:
 $ pip install leverage
 ```
 
+## Custom tasks
+Most of the core functionality of Pynt is still there, but we did introduce the following changes:
```
Comment (exequielrafaela) [inline README.md:34]: @angelofenoglio @diego-ojeda-binbash We should still refer to Pynt here?

**Review by exequielrafaela — APPROVED** [review]

**Pull request merged**